### DISCUSSION TITLE
JobRunr with Integration tests

### DISCUSSION THREAD

**Discussion by Sleepy026** [jobrunr/jobrunr#728] [Q&A]
Hi!!
I am currently running the jobrunr-spring-boot-starter 5.3.3 version in my project.
I have few integration tests and how can I stop JobRunr properly when the tests are finished?
I have a method with @AfterAll annotation where I stop the test containers, 
but the JobRunr is still running and it throws an exception because the DB stopped.

**Comment by rdehuyss** [comment]
There are quite a lot integration tests for JobRunr that show you what's needed.

You should call `backgroundJobServer.stop()`.

**Reply by Sleepy026** [reply]
Can you show me and example, how can you call it from a static method please ?

Edit:

The issue was with the postgres testcontainer, after changed jobrunr's storage to in-memory, the problem solved.
